.chloggen/kafka-topic-context.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: kafkaexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add option to supply destination topic through context. (#34503)
```

```suggestion
note: Add option to supply destination topic through context.
```
The issue (PR in this case) number will be taken from the `issues` section below and appended to the message automatically by `.chloggen`, so we don't need to include it here manually.